pages/dkp/kommander/2.2/dkp-upgrade/index.md

```diff
@@ -1,10 +1,62 @@
 ---
 layout: layout.pug
-navigationTitle: DKP upgrade
-title: How to upgrade DKP
+navigationTitle: Upgrade DKP
+title: How to upgrade DKP via CLI
 menuWeight: 8
-excerpt: Upgrade DKP to the latest version and benefit from the full DKP experience.
+excerpt: Upgrade DKP to the latest version
 beta: false
 ---
 
-This section describes how to upgrade your environment to the latest version of DKP.
+The DKP upgrade represents an important step of your environment's lifecycle, as it ensures that you are up-to-date with the latest features and can benefit from the most recent improvements, enhanced cluster management, and better performance. This section describes how to upgrade your networked, air-gapped or on-prem environment to the latest version of DKP.
```

Comment:
```suggestion
The DKP upgrade represents an important step of your environment's lifecycle, as it ensures that you are up-to-date with the latest features and can benefit from the most recent improvements, enhanced cluster management, and better performance. This section describes how to upgrade your networked, air-gapped, or on-prem environment to the latest version of DKP.
```

Reply:
"Oxford comma" usage - don't skip the last one.
Stealing this, but it's funny:
Let's eat Grandpa!
Let's eat, Grandpa!
Commas save lives.